tests/testthat/test-bootstrap.r

```diff
@@ -69,6 +69,11 @@ test_that("Repeated names for selectInput and radioButtons choices", {
 
 
 test_that("Choices are correctly assigned names", {
+  # Empty character vector
+  expect_identical(
+    choicesWithNames(c("")),
```

Comment:
I think this should test a non-empty string, because `""` isn't a name that will work with Shiny.

Reply:
Note to self: add comment explaining seemingly-pointless test